### PR TITLE
Quality: PowerShell Summit GH-3328 

### LINKED ISSUE #3328: Quality: Refactoring documentation for hgsclient module
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://github.com/Spoonsk/windows-powershell-docs/tree/main/docset/winserver2022-ps/hgsclient

### Summary

Refactor help for all commandlets to fit the __PowerShell-Docs style guide__

### Details

_No response_

### Suggested Fix

_No response_

### PR DESCRIPTION
**Pull request by Spoonsk**
# PR Summary

- Fixes #3328

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by Spoonsk** [comment]
@microsoft-github-policy-service agree

File: docset/winserver2022-ps/hgsclient/ConvertTo-HgsKeyProtector.md [hunk 1]
```diff
@@ -11,54 +11,61 @@ title: ConvertTo-HgsKeyProtector
...
+```powershell
+$VirtualTPM = Get-VMTPM -Name "Shielded Virtual Machine 17"
+$VirtualMachineKeyProtector = $VirtualTPM.KeyProtector
+$KeyProtector = ConvertTo-HgsKeyProtector -Bytes $VirtualMachineKeyProtector
 ```
 
-The first command gets the Trusted Platform Module (TPM) object for the virtual machine named Shielded Virtual Machine 17.
-The command stores the object in the **$VirtualTPM** variable.
+The first command gets the Trusted Platform Module (TPM) object for the virtual machine named
+Shielded Virtual Machine 17. The command stores the object in the **$VirtualTPM** variable.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/ConvertTo-HgsKeyProtector.md:41]:
```suggestion
The first command gets the Trusted Platform Module (TPM) object for the virtual machine named
Shielded Virtual Machine 17. The command stores the object in the `$VirtualTPM` variable.
```

File: docset/winserver2022-ps/hgsclient/ConvertTo-HgsKeyProtector.md [hunk 2]
```diff
@@ -11,54 +11,61 @@ title: ConvertTo-HgsKeyProtector
...
 
-The second command stores the **KeyProtector** property of the object stored in **$VirtualTPM** in the **$VirtualMachineKeyProtector** variable.
+The second command stores the **KeyProtector** property of the object stored in **$VirtualTPM** in
+the **$VirtualMachineKeyProtector** variable.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/ConvertTo-HgsKeyProtector.md:44]:
```suggestion
The second command stores the **KeyProtector** property of the object stored in `$VirtualTPM` in
the `$VirtualMachineKeyProtector` variable.
```

File: docset/winserver2022-ps/hgsclient/ConvertTo-HgsKeyProtector.md [hunk 3]
```diff
@@ -118,21 +131,26 @@ Accept wildcard characters: False
...
 
 ### Microsoft.Management.Infrastructure.CimInstance#MSFT_HgsKeyProtector
+
 This cmdlet returns a key protector.
 
-The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays Windows Management Instrumentation (WMI) objects.
-The path after the pound sign (`#`) provides the namespace and class name for the underlying WMI object.
+The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays
+Windows Management Instrumentation (WMI) objects. The path after the pound sign (`#`) provides the
+namespace and class name for the underlying WMI object.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/ConvertTo-HgsKeyProtector.md:150]:
```suggestion
The **Microsoft.Management.Infrastructure.CimInstance** object is a wrapper class that displays
Windows Management Instrumentation (WMI) objects. The path after the pound sign (`#`) provides the
namespace and class name for the underlying WMI object.
```

File: docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md [hunk 4]
```diff
@@ -20,26 +20,30 @@ Export-HgsGuardian [-InputObject] <CimInstance> [-Path] <String> [<CommonParamet
 ```
 
 ## DESCRIPTION
+
 The **Export-HgsGuardian** cmdlet exports a guardian that contains public keys to an .xml file.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md:24]:
```suggestion
The `Export-HgsGuardian` cmdlet exports a guardian that contains public keys to an `.xml` file.
```

File: docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md [hunk 5]
```diff
@@ -20,26 +20,30 @@ Export-HgsGuardian [-InputObject] <CimInstance> [-Path] <String> [<CommonParamet
...
 
 ## EXAMPLES
 
 ### Example 1: Export a guardian
-```
-PS C:\> Get-HgsGuardian -Name "Guardian11" | Export-HGsGuardian -Path "C:\LocalHGSFiles\Guardian11.xml"
+
+```powershell
+Get-HgsGuardian -Name "Guardian11" | Export-HGsGuardian -Path "C:\LocalHGSFiles\Guardian11.xml"
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md:31]:
```suggestion
Get-HgsGuardian -Name 'Guardian11' |
    Export-HGsGuardian -Path 'C:\LocalHGSFiles\Guardian11.xml'
```

File: docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md [hunk 6]
```diff
@@ -20,26 +20,30 @@ Export-HgsGuardian [-InputObject] <CimInstance> [-Path] <String> [<CommonParamet
...
 ```
 
-This command uses the **Get-HgsGuardian** cmdlet to get the guardian named Guardian11, and then passes the object to the current cmdlet by using the pipeline operator.
-That cmdlet exports the guardian to the specified file.
+This command uses the **Get-HgsGuardian** cmdlet to get the guardian named `Guardian11`, and then
+passes the object to the current cmdlet by using the pipeline operator. That cmdlet exports the
+guardian to the specified file.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md:36]:
```suggestion
This command uses the `Get-HgsGuardian` cmdlet to get the guardian named `Guardian11`, and then
passes the object to the current cmdlet by using the pipeline operator. That cmdlet exports the
guardian to the specified file.
```

File: docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md [hunk 7]
```diff
@@ -20,26 +20,30 @@ Export-HgsGuardian [-InputObject] <CimInstance> [-Path] <String> [<CommonParamet
...
 
 ## PARAMETERS
 
 ### -InputObject
-Specifies the input to this cmdlet. 
+
+Specifies the input to this cmdlet.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Export-HgsGuardian.md:42]:
```suggestion
Specifies the input to this cmdlet.

```

File: docset/winserver2022-ps/hgsclient/Get-HgsAttestationBaselinePolicy.md [hunk 8]
```diff
@@ -26,31 +26,35 @@ Get-HgsAttestationBaselinePolicy [-Console] [-SkipValidation] [<CommonParameters
 ```
 
 ## DESCRIPTION
-The **Get-HgsAttestationBaselinePolicy** cmdlet generates an attestation baseline policy.
-You can use the policy to configure the attestation service.
 
-This cmdlet gets a byte array that represents the attestation baseline policy from the raw data of the last full boot from the Trusted Computing Group log of the Trusted Platform Module (TPM).
+The **Get-HgsAttestationBaselinePolicy** cmdlet generates an attestation baseline policy. You can
+use the policy to configure the attestation service.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsAttestationBaselinePolicy.md:31]:
```suggestion
The `Get-HgsAttestationBaselinePolicy` cmdlet generates an attestation baseline policy. You can use
the policy to configure the attestation service.
```

File: docset/winserver2022-ps/hgsclient/Get-HgsAttestationBaselinePolicy.md [hunk 9]
```diff
@@ -26,31 +26,35 @@ Get-HgsAttestationBaselinePolicy [-Console] [-SkipValidation] [<CommonParameters
...
 
 Be sure to run this cmdlet on a host that is known to have good configuration.
 
 ## EXAMPLES
 
 ### Example 1: Generate a baseline policy
-```
+
+```powershell
 PS C:\> Get-HgsAttestationBaselinePolicy -Path "C:\Logs\AttestationBaselinePolicy001" -Force
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsAttestationBaselinePolicy.md:43]:
```suggestion
Get-HgsAttestationBaselinePolicy -Path 'C:\Logs\AttestationBaselinePolicy001' -Force
```

File: docset/winserver2022-ps/hgsclient/Get-HgsClientConfiguration.md [hunk 10]
```diff
@@ -16,16 +16,20 @@ Gets the configuration of a Host Guardian Service client.
...
-Get-HgsClientConfiguration [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [<CommonParameters>]
+Get-HgsClientConfiguration [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-HgsClientConfiguration** cmdlet get the configuration of the local Host Guardian Service client.
+
+The **Get-HgsClientConfiguration** cmdlet get the configuration of the local Host Guardian Service
+client.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsClientConfiguration.md:26]:
```suggestion
The `Get-HgsClientConfiguration` cmdlet get the configuration of the local Host Guardian Service
client.
```

File: docset/winserver2022-ps/hgsclient/Get-HgsClientConfiguration.md [hunk 11]
```diff
@@ -16,16 +16,20 @@ Gets the configuration of a Host Guardian Service client.
...
 
 ## EXAMPLES
 
 ### Example 1: Get client configuration
-```
+
+```powershell
 PS C:\> Get-HgsClientConfiguration
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsClientConfiguration.md:33]:
```suggestion
Get-HgsClientConfiguration
```

File: docset/winserver2022-ps/hgsclient/Get-HgsClientConfiguration.md [hunk 12]
```diff
@@ -91,20 +102,25 @@ Accept wildcard characters: False
...
 ## OUTPUTS
 
 ### CimInstance#MSFT_HgsClientConfiguration
+
 This cmdlet returns the configuration of a Host Guardian Service client as a **CimInstance** object.
-The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays Windows Management Instrumentation (WMI) objects.
-The path after the pound sign (`#`) provides the namespace and class name for the underlying WMI object.
+The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays
+Windows Management Instrumentation (WMI) objects. The path after the pound sign (`#`) provides the
+namespace and class name for the underlying WMI object.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsClientConfiguration.md:120]:
```suggestion
The **Microsoft.Management.Infrastructure.CimInstance** object is a wrapper class that displays
Windows Management Instrumentation (WMI) objects. The path after the pound sign (`#`) provides the
namespace and class name for the underlying WMI object.
```

File: docset/winserver2022-ps/hgsclient/Get-HgsGuardian.md [hunk 13]
```diff
@@ -21,21 +21,24 @@ Get-HgsGuardian [[-Name] <String[]>] [-CimSession <CimSession[]>] [-ThrottleLimi
 ```
 
 ## DESCRIPTION
+
 The **Get-HgsGuardian** cmdlet gets Host Guardian Service guardian objects from persistent storage.
 For a computer configured in **HostGuardianService** mode, this cmdlet returns no result.
 The **New-HgsKeyProtector** cmdlet requires **HgsGuardian** objects.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsGuardian.md:27]:
```suggestion
The `Get-HgsGuardian` cmdlet gets Host Guardian Service guardian objects from persistent storage.
For a computer configured in `HostGuardianService` mode, this cmdlet returns no result. The
`New-HgsKeyProtector` cmdlet requires **HgsGuardian** objects.
```

File: docset/winserver2022-ps/hgsclient/Get-HgsGuardian.md [hunk 14]
```diff
@@ -21,21 +21,24 @@ Get-HgsGuardian [[-Name] <String[]>] [-CimSession <CimSession[]>] [-ThrottleLimi
...
+Get-HgsGuardian
 ```
 
 This command gets all guardians configured for this computer.
 
 ### Example 2: Get a guardian by using its name
-```
+
+```powershell
 PS C:\> Get-HgsGuardian -Name "Guardian11"
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Get-HgsGuardian.md:42]:
```suggestion
Get-HgsGuardian -Name 'Guardian11'
```

File: docset/winserver2022-ps/hgsclient/Grant-HgsKeyProtectorAccess.md [hunk 15]
```diff
@@ -16,49 +16,54 @@ Grants access to a guardian for a key protector.
...
-Grant-HgsKeyProtectorAccess -KeyProtector <CimInstance> -GuardianFriendlyName <String> [-AllowUntrustedRoot]
- [-AllowExpired] [<CommonParameters>]
+Grant-HgsKeyProtectorAccess -KeyProtector <CimInstance> -GuardianFriendlyName <String>
+ [-AllowUntrustedRoot] [-AllowExpired] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
+
 The **Grant-HgsKeyProtectorAccess** cmdlet grants a Host Guardian Service guardian access to a key protector.
 This operation requires the private signing key of the owner of the key protector.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Grant-HgsKeyProtectorAccess.md:34]:
```suggestion
The `Grant-HgsKeyProtectorAccess` cmdlet grants a Host Guardian Service guardian access to a key
protector. This operation requires the private signing key of the owner of the key protector.
```

File: docset/winserver2022-ps/hgsclient/Grant-HgsKeyProtectorAccess.md [hunk 16]
```diff
@@ -16,49 +16,54 @@ Grants access to a guardian for a key protector.
...
-PS C:\> $Owner = Get-HgsGuardian -Name "Guardian06"
-PS C:\> $Guardian01 = Get-HgsGuardian -Name "Guardian11"
-PS C:\> $KeyProtector = New-HgsKeyProtector -Owner $Owner 
-PS C:\> Grant-HgsKeyProtectorAccess -KeyProtector $KeyProtector -Guardian $Guardian01
+
+```powershell
+$Owner = Get-HgsGuardian -Name "Guardian06"
+$Guardian01 = Get-HgsGuardian -Name "Guardian11"
+$KeyProtector = New-HgsKeyProtector -Owner $Owner
+Grant-HgsKeyProtectorAccess -KeyProtector $KeyProtector -Guardian $Guardian01
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Grant-HgsKeyProtectorAccess.md:44]:
```suggestion
$Owner        = Get-HgsGuardian -Name "Guardian06"
$Guardian01   = Get-HgsGuardian -Name "Guardian11"
$KeyProtector = New-HgsKeyProtector -Owner $Owner
Grant-HgsKeyProtectorAccess -KeyProtector $KeyProtector -Guardian $Guardian01
```

File: docset/winserver2022-ps/hgsclient/Grant-HgsKeyProtectorAccess.md [hunk 17]
```diff
@@ -128,15 +137,18 @@ Accept wildcard characters: False
...
 
 ## OUTPUTS
 
 ### CimInstance#MSFT_HgsKeyProtector
-The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays Windows Management Instrumentation (WMI) objects.
-The path after the pound sign (`#`) provides the namespace and class name for the underlying WMI object.
+
+The `Microsoft.Management.Infrastructure.CimInstance` object is a wrapper class that displays
+Windows Management Instrumentation (WMI) objects. The path after the pound sign (`#`) provides the
+namespace and class name for the underlying WMI object.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hgsclient/Grant-HgsKeyProtectorAccess.md:151]:
```suggestion
The **Microsoft.Management.Infrastructure.CimInstance** object is a wrapper class that displays
Windows Management Instrumentation (WMI) objects. The path after the pound sign (`#`) provides the
namespace and class name for the underlying WMI object.
```

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**